src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
> Shouldn't Markdown parsing be handled by markdown extra, kind of in "collaboration" with description extra? markdown should be applied after description if one want Markdown description.

Why not. Ping @HugoGiraudel 

> Note isFile testing actual file existence instead of isPath "dumb" regex matching, allowing for non-Markdown descriptions.)

Not sure to get the sentence here...